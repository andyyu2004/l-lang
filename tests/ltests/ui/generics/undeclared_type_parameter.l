struct S {
    t: T
}

fn main() -> int {
    0
}


