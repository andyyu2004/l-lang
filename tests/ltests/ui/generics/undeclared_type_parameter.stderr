error: unresolved type `T`
  ┌─ undeclared_type_parameter.l:2:8
  │
2 │     t: T
  │        ^

1 error emitted
